.travis.yml

```diff
@@ -73,6 +74,12 @@ matrix:
       addons:
         hosts:
           - mysql2gem.example.com
+    - rvm: 2.4
+      # Mark
```

Comment:
Who is Mark?

Reply:
"Mark" is not people's name in this case. I want to describe `env: DOCKER=centos` is just used to identify this test case from Travis test page. But I am going to remove the line.